coralproject/talk
# apply custom CSS to admin panel

## What does this PR do?
Applies styles from custom CSS file to the admin panel.
See https://github.com/coralproject/talk/issues/1780


## How do I test this PR?
1. View admin panel
2. Create .css file that overrides styles on an existing element on the admin panel
3. Update custom admin CSS url to reference .css file from step 2
4. Refresh admin panel to see updated styles
